mace/calculators/foundations_models.py

```diff
@@ -121,6 +121,72 @@ def mace_mp(
     return calc
 
 
+def mace_off(
+    model: Union[str, Path] = None,
+    device: str = "",
+    default_dtype: str = "float32",
```

Comment:
The default dtype does not match the doc string.

Reply:
good catch, I am updating it.